### PR TITLE
Equations added

### PR DESCRIPTION
**Pull request by kschwarzberg**
This partially addresses issue #321 by adding these equations: ace, enspie, heip_e, fisher_alpha, kempton_taylor_q, mcintosh_e, menhinick, shannon, simpson, and strong. In addition, variable definitions were added to berger_parker_d and to brillouin_d.

### CODE REVIEW

**Comment by Jorge-C** [comment]
What needs to be done before this PR is ready, apart from solving merge conflicts?

I think the docstrings should be raw (`r"""..."""`) and remove double backslashes for clarity in the plain text view of the docstrings. Anything else you can think of?

**Comment by ebolyen** [comment]
We still need to review the equations I believe.

**Comment by jairideout** [comment]
Thanks for the reminder @Jorge-C, this dropped off my radar. I'll review this week.

**Comment by jairideout** [comment]
@gregcaporaso and I chatted and I'll issue a new pull request, containing the commits here, and address the remaining comments.

File: skbio/diversity/alpha/base.py [hunk 1]
```diff
@@ -71,11 +75,17 @@ def berger_parker_d(counts):
...
 def brillouin_d(counts):
-    """Calculate Brillouin index of alpha diversity, which is defined as:
+    """Calculate Brillouin index of alpha diversity.
+
+    This is calculated as follows:
 
     .. math::
 
-       HB = \\frac{\\ln N!-\\sum^5_{i=1}{\\ln n_i!}}{N}
+       HB = \\frac{\\ln N!-\\sum^s_{i=1}{\\ln n_i!}}{N}
```
Comment (jairideout) [inline skbio/diversity/alpha/base.py]: The equation and code should match. The code uses `gammaln(n+1)`, which is equivalent to `ln(abs(gamma(n+1)))`, where `gamma(n+1)` is equivalent to `n!` (see scipy docs [here](http://docs.scipy.org/doc/scipy-0.15.1/reference/generated/scipy.special.gammaln.html#scipy.special.gammaln) and [here](http://docs.scipy.org/doc/scipy-0.15.1/reference/generated/scipy.special.gamma.html#scipy.special.gamma) for details).

File: skbio/diversity/alpha/base.py [hunk 2]
```diff
@@ -45,6 +45,10 @@ def berger_parker_d(counts):
 
        d = \\frac{N_{max}}{N}
 
+    :math:`N_{max}` is defined as the number of individuals in the most
+    abundant OTUs, and :math:`N` is defined as the total number of individuals
```
Comment (jairideout) [inline skbio/diversity/alpha/base.py]: fixed

File: skbio/diversity/alpha/base.py [hunk 3]
```diff
@@ -289,7 +307,17 @@ def esty_ci(counts):
 
 
 def fisher_alpha(counts):
-    """Calculate Fisher's alpha.
+    """Calculate Fisher's alpha, a metric of diversity.
```
Comment (jairideout) [inline skbio/diversity/alpha/base.py]: fixed

File: skbio/diversity/alpha/base.py [hunk 4]
```diff
@@ -399,8 +436,19 @@ def heip_e(counts):
 def kempton_taylor_q(counts, lower_quantile=0.25, upper_quantile=0.75):
     """Calculate Kempton-Taylor Q index of alpha diversity.
 
-    Estimates the slope of the cumulative abundance curve in the interquantile
-    range. By default, uses lower and upper quartiles, rounding inwards.
+    The Q index is defined as:
+
+    .. math::
+
+      Q = \\frac{\\frac{1}{2}n_{R1}+\\sum_{R1+1}^{R2-1}n_r+\\frac{1}{2}n_{R2}}
```
Comment (jairideout) [inline skbio/diversity/alpha/base.py]: Reverting the changes to this one until #1035 is resolved.

File: skbio/diversity/alpha/base.py [hunk 5]
```diff
@@ -759,6 +828,14 @@ def robbins(counts):
 def shannon(counts, base=2):
     """Calculate Shannon entropy of counts (H), default in bits.
 
+    .. math::
+
+      H = -\\sum_{i=1}^s\\left(p_i\\log_2 p_i\\right)
+
+    Where :math:`p_i` is the proportion of individuals in the ith OTUs,
```
Comment (jairideout) [inline skbio/diversity/alpha/base.py]: fixed

File: skbio/diversity/alpha/base.py [hunk 6]
```diff
@@ -884,6 +968,17 @@ def singles(counts):
 def strong(counts):
     """Calculate Strong's dominance index (Dw).
 
+    Strong's dominance index is defined as:
```
Comment (jairideout) [inline skbio/diversity/alpha/base.py]: fixed

**Comment by jairideout** [comment]
Closing in favor of #1037, which contains the commits here and has updates based on this PR's code review.

**Pull request closed without merging**